apps/portal/components/containers/dashboardLayout/elements/SidebarMenuItems/SidebarMenuItem.tsx

```
@@ -9,32 +12,129 @@ interface ISidebarMenuItemProps {
+            )}
+          </>
+        ) : (
           <div
             className={classNames(
-              styles.contentWrapper,
+              styles.wrapper,
               active && styles.active,
+              className,
             )}
+            onClick={onClick}
+            tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)